Gemfile.lock

```diff
@@ -295,7 +295,7 @@ GEM
     descendants_tracker (0.0.4)
       thread_safe (~> 0.3, >= 0.3.1)
     diff-lcs (1.4.4)
-    docile (1.3.2)
+    docile (1.3.5)
```

Comment:
Could you also note the gem dependencies of simplecov that had to be updated in the PR description please. Looks like docile, simplecov-html and simplecov_json_formatter